docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
+| workqueue_work_duration_seconds               | Histogram | How long in seconds processing an item from workqueue takes.                                                                                                                                                                         |
+| workqueue_unfinished_work_seconds             | Gauge     | How many seconds of work has been done that is in progress and hasn't been observed by work_duration. Large values indicate stuck threads. One can deduce the number of stuck threads by observing the rate at which this increases. |
+| workqueue_longest_running_processor_seconds   | Gauge     | How many seconds has the longest running processor for workqueue been running.                                                                                                                                                       |
+| workqueue_retries_total                       | Counter   | Total number of retries handled by workqueue.                                                                                                                                                                                        |
+| rest_client_requests_total                    | Counter   | Number of HTTP requests, partitioned by status code, method, and host.                                                                                                                                                               |
+| controller_runtime_reconcile_total            | Counter   | Total number of reconciliations per controller.                                                                                                                                                                                      |
+| controller_runtime_reconcile_errors_total     | Counter   | Total number of reconciliation errors per controller.                                                                                                                                                                                |
+| controller_runtime_reconcile_time_seconds     | Histogram | Length of time per reconciliation per controller.                                                                                                                                                                                    |
+| controller_runtime_max_concurrent_reconciles  | Gauge     | Maximum number of concurrent reconciles per controller.                                                                                                                                                                              |
+| controller_runtime_active_workers             | Gauge     | Number of currently used workers per controller.                                                                                                                                                                                     |
```

Comment:
These 4 ones are defined in (https://github.com/kubernetes-sigs/controller-runtime/blob/1e4d87c9f9e15e4a58bb81909dd787f30ede7693/pkg/internal/controller/metrics/metrics.go#L26-L63)

Reply:
👍🏻